### PR TITLE
CASMPET-5066 - Drive location doc

### PR DESCRIPTION
**Pull request by cdelatte-hpe**
## Summary and Scope
- utilizes ledmon/ledctl for drive led beacon

## Issues and Related PRs

* Resolves CASMPET-5066

### Tested on:

  * mug

### Test description:

Enabled and disabled the LED locator beacon on drives in the mug environment.  DHCW confirmed the drive beacons were turned on and off.

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)? n/a
- Were continuous integration tests run? If not, why? n/a
- Was upgrade tested? If not, why? n/a
- Was downgrade tested? If not, why? n/a
- Were new tests (or test issues/Jiras) created for this change? n/a

## Risks and Mitigations

n/a

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by bklei — APPROVED** [review]

File: operations/node_management/NCN_Identify_Drives_Using_ledctl.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# NCN Drive Identification
+
+Basic usage for the ledmon/ledctl software for drive indentification via the drive leds.
+
+## Usage
+
+Turn on led locator beacon
```
Comment (nrockershousen) [inline operations/node_management/NCN_Identify_Drives_Using_ledctl.md:7]: - Add ":" to the end of this line.
- led -> LED

File: operations/node_management/NCN_Identify_Drives_Using_ledctl.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+```bash
+ledctl locate=/dev/<drive>
+```
+
+Turn off led locator beacon
```
Comment (nrockershousen) [inline operations/node_management/NCN_Identify_Drives_Using_ledctl.md:13]: - Add ":" to the end of this line.
- led -> LED

File: operations/node_management/NCN_Identify_Drives_Using_ledctl.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# NCN Drive Identification
```
Comment (nrockershousen) [inline operations/node_management/NCN_Identify_Drives_Using_ledctl.md:1]: The file name should match the title.

NCN_Drive_Identifiction.md

File: operations/node_management/NCN_Identify_Drives_Using_ledctl.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+Basic usage for the ledmon/ledctl software for drive indentification via the drive leds.
```
Comment (nrockershousen) [inline operations/node_management/NCN_Identify_Drives_Using_ledctl.md:3]: Change to:

Use the `ledmon`/`ledctl` software to identify NCN drives via the drive LEDs.

**Comment by nrockershousen** [comment]
An entry for this new file should be added under the Node Management header in the operations/index.md file.

**Pull request merged**